components/schema-migrator/migrations/director/20230119093834_introduce-cert-subject-mapping-entity.up.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE cert_subject_mapping (
+    id UUID PRIMARY KEY CHECK (id <> '00000000-0000-0000-0000-000000000000'),
+    subject VARCHAR(256) NOT NULL,
+    CONSTRAINT cert_mapping_subject_unique UNIQUE (subject),
+    consumer_type VARCHAR(256) NOT NULL CHECK ( consumer_type IN ('Runtime', 'Integration System', 'Application', 'Super Admin"', 'Business Integration', 'Technical Client', 'global')),
+    internal_consumer_id UUID CHECK (internal_consumer_id <> '00000000-0000-0000-0000-000000000000'),
```

Comment:
Why we require the internal_consumer_id to be UUID, seems unnecessary restriction to me.